.github/workflows/publish-linux-packages.yml

```diff
@@ -0,0 +1,126 @@
+name: Publish Linux Packages
+
+on:
+  workflow_call:
+    inputs:
+      ziti-version:
+        description: generated by 'ziti-ci get-current-version'
+        type: string
+        required: true
+
+# cancel older, redundant runs of same workflow on same branch
+concurrency:
```

Comment:
We experienced a deadlock with the Jenkins smoke test reusable workflow. It may be necessary to move this concurrency group to the job level or remove it and rely on the calling workflow's concurrency group.